solr/core/src/java/org/apache/solr/cloud/OverseerTaskProcessor.java

```diff
@@ -149,7 +150,15 @@ public OverseerTaskProcessor(
     this.zkStateReader = zkStateReader;
     this.myId = myId;
     this.stats = stats;
-    this.selector = selector;
+    this.overseerCollectionMessageHandler =
```

Comment:
This needs to be closed.  The test failures are related to this; threads leak from it thus it wasn't closed.

Reply:
Okay, got it. Thanks